rust-runtime/aws-smithy-client/src/builder.rs

```diff
@@ -135,7 +135,7 @@ impl<M, R> Builder<(), M, R> {
         let with_https = |b: Builder<_, M, R>| b.rustls_connector(connector_settings);
         // If we are compiling this function & rustls is not enabled, then native-tls MUST be enabled
         #[cfg(not(feature = "rustls"))]
-        let with_https = |b: Builder<_, M, R>| b.native_tls_connector();
+        let with_https = |b: Builder<_, M, R>| b.native_tls_connector(connector_settings);
```

Comment:
A little bug that sneaked through since we use `--all-features` for testing, I assume.

Reply:
Hmmm... Looks like we forgot to add `cargo hack` to `aws-smithy-client`'s `additional-ci` script. Could you add that so this doesn't happen again?

Example: https://github.com/awslabs/smithy-rs/blob/0fb5ccb81446b76cb20dd26c0e39e0cbc3965b8d/rust-runtime/aws-smithy-eventstream/additional-ci#L15

Reply:
Done!